src/passes/Print.cpp

```diff
@@ -2698,6 +2701,18 @@ void PrintSExpression::printDebugLocation(Expression* curr) {
         doIndent(o, indent);
       }
     }
+
+    auto iter2 = currFunction->codeAnnotations.find(curr);
```

Comment:
Can we rename the iterators `locIter` and `annotIter` or similar? Alternatively, we could limit the scope of the `iter` variables by using `if (auto iter = ...; iter != ...) { }`.

Reply:
Done.